text/0000-unsafe-statics.md

```diff
@@ -0,0 +1,98 @@
+- Feature Name: unsafe_static
+- Start Date: 2020-05-29
+- RFC PR: [rust-lang/rfcs#0000](https://github.com/rust-lang/rfcs/pull/0000)
+- Rust Issue: [rust-lang/rust#0000](https://github.com/rust-lang/rust/issues/0000)
+
+# Summary
+[summary]: #summary
+
+Replace static muts with unsafe statics, which are error prone.
+
+# Motivation
+[motivation]: #motivation
+
+Since before 1.0, Rust has had a feature called `static mut`, which allows users to define statics
+to which mutable references can be taken. Actually taking a mutable reference is unsafe. However,
+this feature is incredibly footgunny and almost impossible to use correctly, since creating a
+mutable reference to shared memory is undefined behavior, even if you never dereference that
```

Comment:
When you say 'shared memory', do you mean 'memory that already has a immutable/mutable reference to it'? The current wording makes it sound as though created *any* mutable reference is undefined behavior, since a static is always 'shared' by virtue of having global scope.